pallets/proxy-keystore/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = 'pallet-proxy-keystore'
```

Comment:
Maybe it would make more sense to just call this the `keystore` pallet? Since there's nothing in the pallet that relates to proxies, it's just storing keys for accounts, not for proxies.

Reply:
Sure, I just copied the name we had in the spec :D gonna change it.